pageserver/src/consumption_metrics.rs

```diff
@@ -202,543 +72,272 @@ pub async fn collect_metrics(
         },
     );
 
+    let final_path: Arc<PathBuf> = Arc::new(local_disk_storage);
```

Comment:
nit: `final_path` is a bit of a generic name, maybe `stats_cache_path` or something like that?

Reply:
I was thinking more like `path`, this is a remnant of me trying to juggle multiple paths. Will change.

Reply:
Next push 658956f51